packages/common/src/authorization/organizationMemberAbility.ts

```diff
@@ -38,28 +39,104 @@ const applyOrganizationMemberStaticAbilities: Record<
             organizationUuid: member.organizationUuid,
         });
     },
-    viewer(member, { can }) {
+    viewer(member, { can }, groupMemberships = []) {
         applyOrganizationMemberStaticAbilities.member(member, { can });
+
+        // Dashboards
         can('view', 'Dashboard', {
             organizationUuid: member.organizationUuid,
             isPrivate: false,
         });
-        can('view', 'SavedChart', {
+
+        can('view', 'Dashboard', {
```

Comment:
These rules are basically repeated for:

- dashboards
- charts
- spaces

for both the project member ability **and** the organization member ability. So look for the patterns!